cfgov/unprocessed/css/molecules/equal-height-columns.less

```diff
@@ -0,0 +1,60 @@
+/* topdoc
+  name: Equal height columns
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-equal-height_container">
+            <div class="m-equal-height_col">
+                <div class="m-equal-height_content">
+                	Equal height box contents
+            	</div>
+            </div>
+            <div class="m-equal-height_col">
+                <div class="m-equal-height_content">
+                	Equal height box contents
+                	<div class="m-equal-height_footer">
+                		Bottom aligned footer
+                	</div>
+            	</div>
+            </div>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-equal-height_container
+            .m-equal-height_col
+              .m-equal-height_content
+              	.m-equal-height_footer
+  tags:
+    - cfgov-molecules
+  notes:
+    - No support for equal-height columns below IE10.
+      Can be used in conjunction with content-layout
+      classes to provide column widths and layout for
+      browsers that do not support flexbox. 
+*/
+
+.m-equal-height_container {
```

Comment:
This feels like it should be an extension to cf-layout, and either a utility or a layout class rather than an atomic component. It could be used anywhere we have things in rows/columns and it wouldn't matter if the parent or children were molecules or organisms